app/src/main/res/drawable/ic_new_pin.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="10dp"
+    android:height="10dp"
+    android:viewportWidth="10"
+    android:viewportHeight="10">
+    <path
+        android:fillColor="@color/top_site_pin_icon_background_tint"
+        android:pathData="M8.05,3.055H6.6L6.156,2.89C6.156,2.89 5.19,1.926 4.612,1.349L5.106,0.855C5.302,0.659 5.302,0.342 5.106,0.147C4.91,-0.049 4.593,-0.049 4.398,0.147L3.689,0.855L0.147,4.398C-0.049,4.594 -0.049,4.911 0.147,5.106C0.343,5.302 0.66,5.302 0.855,5.106L1.349,4.612C1.93,5.194 2.904,6.17 2.904,6.17L3.054,6.597L3.055,8.049C3.055,8.425 3.279,8.76 3.626,8.904C3.742,8.952 3.862,8.975 3.981,8.975C4.221,8.975 4.457,8.881 4.634,8.703L6.314,7.023L9.144,9.853C9.34,10.049 9.657,10.049 9.852,9.853C10.048,9.657 10.048,9.34 9.852,9.145L7.022,6.315L8.702,4.635C8.968,4.369 9.046,3.974 8.903,3.627C8.761,3.279 8.425,3.055 8.05,3.055Z" />
```

Comment:
Looking at the pathData, I think you exported the 10x10 SVG vector. I know you can also export it as 12x12 from my examination. I am not quite sure what is exactly intended. So, perhaps the best thing we can do here is have @brampitoyo export it and we use the provided pathData. This also part of the larger size question above.